client/src/components/manageProjects/utilities/validateEventForm.js

```diff
@@ -1,19 +1,24 @@
 import validator from 'validator';
-import { isAnySubstring } from './../../../utils/stringUtils.js';
-import { useProjectsStore } from '../../../store/projectStore.js';
+import { isWordInArrayInString } from './../../../utils/stringUtils.js';
 
-const validateEventForm = (vals) => {
+const validateEventForm = (vals, projectToEdit) => {
```

Comment:
`CreateNewEvent()` fails because of this line

One solve is to have projectToEdit default to null `(vals, projectToEdit = null) =>`